EIPS/eip-x.md

```diff
@@ -25,61 +25,69 @@ The ERC-4626 Tokenized Vaults standard has helped to make yield-bearing tokens m
+An important Vault inequality is that following a Request(s), the cumulative requested quantity MUST be more than `pendingDepositRequest + maxDeposit - claimed`. The inequality may come from fees or other state transitions outside implemented by Vault logic such as cancellation of a Request, otherwise this would be a strict equality.
 
-Requests MUST NOT skip or otherwise short-circuit the claim step. In other words, to initiate and claim a request, a user MUST call both request* and the corresponding claim function separately, even in the same block.
+Requests MUST NOT skip or otherwise short-circuit the Claim state. In other words, to initiate and claim a Request, a user MUST call both request* and the corresponding Claim function separately, even in the same block.
 
-For asynchronous request vaults, the exchange rate between shares and assets including fees and yield is up to the vault implementation. In other words, pending redemption requests MAY NOT be yield bearing and MAY NOT have a fixed exchange rate.
+For asynchronous Vaults, the exchange rate between `shares` and `assets` including fees and yield is up to the Vault implementation. In other words, pending redemption Requests MAY NOT be yield bearing and MAY NOT have a fixed exchange rate.
+
+The view function relationship is summarized in the below table where the Request input amount is 1000 and pending/max view function input is the `operator`:
+
+| **State**   | **Pending View Output** | **Max View Output** |
+|-------------|-------------------------|--------------------|
+| Pending     | 1000                    | 0                  |
+| Claimable   | 0                       | 1000               |
+| Claimed     | 0                       | 0                  |
 
 ### Methods
 
 #### requestDeposit
 
-Locks `assets` from `msg.sender` into the Vault and submits a Request for asynchronous `deposit/mint`. This places the Request in Pending state, with a corresponding increase in `pendingDepositRequest` for the amount `assets`.
+Transfers `assets` from `msg.sender` into the Vault and submits a Request for asynchronous `deposit/mint`. This places the Request in Pending state, with a corresponding increase in `pendingDepositRequest` for the amount `assets`.
 
-When the Request is claimable, `maxDeposit` and `maxMint` will be increased, and `deposit` or `mint` can be called by `operator` to receive `shares`. A Request MAY transition straight to claimable state but MUST NOT skip the claimable state.
+When the Request is Claimable, `maxDeposit` and `maxMint` will be increased for the case where the `receiver` input is the `operator`. `deposit` or `mint` can subsequently be called by `operator` to receive `shares`. A Request MAY transition straight to Claimable state but MUST NOT skip the Claimable state.
```

Comment:
"A Request MAY transition straight to Claimable state but MUST NOT skip the Claimable state"

if it requests can go straight to the Claimable state (no gating from protocol), what is different from the synchronous flow? in that case shouldn't the `requestDeposit` method not be called over normal `deposit`?

Reply:
The point is that in cases like the Lido quick withdraw module, its still always a two step process because the default case is async. Protocols may implement a sync liquidity buffer

Reply:
ahh got it, makes sense